src/components/settings/forms/Sector.vue

```diff
@@ -130,12 +130,38 @@
             class="form-section__inputs--fill-w"
           />
         </section>
+        <section class="form-section__buttons">
+          <unnnic-button
+            :text="$t('delete_sector')"
+            type="warning"
+            iconLeft="delete"
+            size="small"
+            @click="openModalDelete = true"
+          />
+        </section>
+        <unnnic-modal-next
```

Comment:
`unnnic-modal-next` is a deprecated component. Use `unnnic-modal` instead ([Story of Unnnic Modal](https://unnnic.dev.cloud.weni.ai/?path=/story/example-modal--normal))